pipelines/controller/src/main/java/org/openmrs/analytics/metrics/FlinkJobListener.java

```diff
@@ -46,6 +46,10 @@ public void onJobExecuted(
       return;
     }
 
+    if (jobExecutionResult == null) {
```

Comment:
Please revert this change as we have renamed packages and it is also included in #813.